src/demo/CistromeExplorer.js

```diff
@@ -313,13 +314,19 @@ export default function App() {
                 </div>
             </div>
 
-            <div className="cistrome-explorer">
-                <div className="container">
-                    <CistromeHGW 
+            <div className="visualization-container">
+                <div className="visualization">
+                    <HiGlassWithMetadata 
                         viewConfig={demos[selectedDemo].viewConfig}
                         options={demos[selectedDemo].options}
                         onViewConfigChange={onViewConfigChange}
                     />
+                    <CistromeToolkit
```

Comment:
Do you think we need to remove `PubSub` entirely from `CistromeToolkit`, and instead pass variables (e.g., visibility) and functions (e.g., onIntervalSearch) to `CistromeToolkit`?

Reply:
Yes I think that using props is a more natural approach than PubSub when using react

Reply:
Okay, will update that!